homework/content.js

```diff
@@ -27,11 +24,25 @@
       p.textContent = passage;
       buff.appendChild(p);
     });
-    _wrapper.appendChild(h);
-    _wrapper.appendChild(buff);
+    this._wrapper.appendChild(h);
+    this._wrapper.appendChild(buff);
+  },
+
+  handleEvent(event) {
+    switch(event.type) {
+        case 'note-open':
+          var note = event.detail;
+          this.resetWrapper();
+          this.drawNote(note);
+          break;
+    }
+  },
+
+  start() {
+    this._wrapper = document.querySelector('#note-content-wrapper');
+    window.addEventListener('note-open', this);
```

Comment:
Good. Binding `this` as `listener` for the API is better than an anonymous function.